DESCRIPTION

```diff
@@ -1,48 +1,39 @@
-Package: PACTA.analysis
-Title: Helpers for 'PACTA_analysis'
+Package: PACTA.analysis.dependencies
+Title: Install Depdendencies of PACTA_analysis
 Version: 0.0.0.9000
 Authors@R: 
-    c(person(given = "Mauro",
-             family = "Lepore",
-             role = c("aut", "ctr", "cre"),
-             email = "maurolepore@gmail.com",
-             comment = c(ORCID = "0000-0002-1986-7988")),
-      person(given = "Clare",
-             family = "Murray",
-             role = "aut",
-             email = "clare@2degrees-investing.org"),
-      person(given = "Jacob",
-             family = "Kastl",
-             role = "aut",
-             email = "jacob.kastl@gmail.com"),
       person(given = "2 Degrees Investing Initiative",
-             role = c("cph", "fnd"),
+             role = c("aut", "ctr", "cre"),
              email = "contact@2degrees-investing.org"))
-Description: For more information visit
-    <https://2degrees-investing.org/>.
+Description: We abuse this DESCRIPTION file to make it easy for Docker to 
```

Comment:
Blocking: If you're openly admitting that you're abusing a design pattern, it means you're probably using the wrong design pattern.